docs/developers/styleguide.rst

```diff
@@ -210,7 +210,6 @@ Like in :pep:`8`, but:
     from lxml.html import fromstring
 
     from translate.filters import checks
-    from translate.storage import versioncontrol
```

Comment:
@nijel This was actually an example on the styleguide. So please replace this with something that fits the example, like `from translate.storage import base`

Reply:
Done in https://github.com/translate/translate/pull/4138